types/quill/index.d.ts

```diff
@@ -42,6 +43,7 @@ export interface KeyboardStatic {
 }
 
 export interface ClipboardStatic {
+    convert(html: string|Blot): DeltaStatic;
```

Comment:
am i looking at the wrong place [here](https://github.com/quilljs/quill/blob/4a6bd6a0fb77078bb280764259a78f7f28254e4a/modules/clipboard.js#L79)?

the method signature shows it takes an object with properties `html` and/or `text`. Not a string or `Blot` (unless a blot has those properties...).

presumably it somehow worked when you tried it so maybe im in the wrong part of the source?

Reply:
yes, the last release is 1.3.6, I think you are looking at the master branch

https://github.com/quilljs/quill/blob/1.3.6/modules/clipboard.js

Reply:
1.3.6  is the latest release 
https://github.com/quilljs/quill/releases